src/jukebox/jukebox/version.py

```diff
@@ -1,14 +1,14 @@
 
 VERSION_MAJOR = 3
-VERSION_MINOR = 4
+VERSION_MINOR = 5
 VERSION_PATCH = 0
-VERSION_EXTRA = ""
+VERSION_EXTRA = "alpha"
```

Comment:
Maybe something like “dev”?

Reply:
'alpha' or 'beta' are the Semantic Versioning defaults and a common standard.
But we could change it to 'develop' to get the reference to the branch.

Reply:
In general I’m fine with either one.